examples/examples_go_test.go

```diff
@@ -53,6 +53,11 @@ func TestPulumiLabelsSecretGo(t *testing.T) {
 }
 
 func getGoBaseOptions(t *testing.T) integration.ProgramTestOptions {
+	if os.Getenv("PULUMI_GO_DEP_ROOT") == "" {
+		depRoot := getCwd(t) + "/../../"
+		err := os.Setenv("PULUMI_GO_DEP_ROOT", depRoot)
+		require.NoError(t, err)
+	}
```

Comment:
Can you point to where we consume this env var? We don't set it in .ci-mgmt and I can't find it used anywhere.

Reply:
Defined here: https://github.com/pulumi/pulumi-gcp/blob/48e6413eded88e0a93c5ecf982745e669d84ab71/.github/workflows/run-acceptance-tests.yml#L28

Used in https://github.com/pulumi/pulumi/blob/3b9449325e484845962e2313bfcff19ea264cc4b/pkg/testing/integration/program.go#L2446